### DISCUSSION TITLE
How to add multiple hostIP of VM in action metadata and I need create IP as to be aliased as same action as aliased.

### DISCUSSION THREAD

**Discussion by harikalyank** [StackStorm/st2#5245] [Packs]
The below is the action metadata and executing same from slack "!dockerps vmpubip" .So i need add vmpubip also be aliased. Please help me to do this.

---
name: status
pack: docker_status
runner_type: "remote-shell-script"
description: "Checking docker continer status on FPM VM's"
enabled: true
entry_point: "docker_status.sh"
parameters:
 ip:
   type: string
   #default:
    # - 65.0.194.82
   required: true
 hosts:
   default: '{{ip}}'
 private_key:
   immutable: true
   default: "/home/stanley/.ssh/test"
 username:
   immutable: true
   default: "centos"
 timeout:
   type: "integer"
   default: 60
#alias.yaml:

[centos@ip-172-31-10-207 packs]$ cat docker_status/aliases/alias.yaml
---
name: dockerps
pack: docker_status
action_ref: "docker_status.status"
description: "Verifying the docker container status on Xdge VMs"
formats:
  - "dockerps {{ip}}"

**Comment by blag** [comment]
Please use [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/) to format code and YAML blocks.

Additionally, you may want to consult the [JSONSchema documentation for specifying arrays](https://json-schema.org/understanding-json-schema/reference/array.html).

If you still have questions, you might find more help in the StackStorm [Forums](https://forum.stackstorm.com) and the [Slack community](https://stackstorm-community.slack.com).
